### ISSUE TITLE
Is it lack of one param of get_schema in onnx_cpp2py_export.defs.pyi

### ISSUE THREAD

**Issue by fumihwh** [onnx/onnx#766]
onnx_cpp2py_export.defs.pyi
`def get_schema(op_type: Text) -> OpSchema: ...`

onnx_cpp2py_export.cc
```
defs.def("get_schema", [](const std::string& op_type,
                            const int& maxInclusiveVersion) -> OpSchema {
```
Should we add maxInclusiveVersion into onnx_cpp2py_export.defs.pyi?

**Comment by linkerzhang** [comment]
please also refer to https://github.com/onnx/onnx/pull/764 to ensure domain is added.

**Comment by askhade** [comment]
fix merged #764
